src/components/shared/eventcard/eventcard.css.js

```diff
@@ -0,0 +1,94 @@
+import styled from 'styled-components';
+import { grid, space, color, typography, layout } from 'styled-system';
+import Text from 'components/shared/text';
+import Box from 'components/shared/box';
+
+// style-system default breakpoints
+// const breakpoints = ['40em', '52em', '64em']
+export const Article = styled.article`
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-template-rows: auto auto 1fr;
+  grid-template-areas:
+    'date title'
+    'image image'
+    'rsvp rsvp';
+
+  @media screen and (min-width: 64em) {
```

Comment:
Would strongly prefer to encode breakpoints in variables for consistency.

Reply:
This should be all set now on this branch. See 60a5fdf.